specification/e-referrals-service-api.yaml

```diff
@@ -537,9 +537,133 @@ paths:
           $ref: 'components/schemas/responses/Forbidden.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /STU3/v1/Binary:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        
+        Use this endpoint to upload a file containing clinical information to e-RS ready to be associated with a referral.
+
+        ## Pre-requisites
+        In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
+          - `SERVICE_PROVIDER_CLINICIAN`
+
+        You can only upload file formats that are supported by e-RS. Currently, the supported file formats are:
+
+        | File type          | File extension(s)           |
+        | ------------------ | --------------------------- |
+        | Plain Text         | `.txt`                      |
+        | HTML Text          | `.html,` `.htm`             |
+        | PDF                | `.pdf`                      |
+        | XML Text           | `.xml`                      |
+        | RTF Text           | `.rtf`, `.rtx`              |
+        | Basic Audio        | `.pdf`                      |
```

Comment:
PDF -> basic audio .. really ?

Reply:
Seem to be a copy&paste mistake. I'll fix it